backend/typescript/types.ts

```diff
@@ -89,6 +88,8 @@ export type CreateCampDTO = Omit<
 >;
 export type CreateCamperDTO = Omit<CamperDTO, "id">;
 
+export type UpdateCamperDTO = Omit<CamperDTO, "id">;
```

Comment:
we should omit registrationDate here as well since we can't update that
```suggestion
export type UpdateCamperDTO = Omit<CamperDTO, "id" | "registrationDate">;
```
```suggestion
export type UpdateCamperDTO = Omit<CamperDTO, "id">;
```